vectorstores/cloudsql/vectorstore.go

```diff
@@ -136,3 +221,18 @@ func (vs *VectorStore) IsValidIndex(ctx context.Context, indexName string) (bool
 
 	return indexnameFromDb == indexName, nil
 }
+
+func vectorToString(vec []float32) string {
```

Comment:
can we get a comment on this function? Not super clear why it's needed

Reply:
changed it to use pgvector new vcetor